frontend/.eslintrc.json

```diff
@@ -1,13 +1,30 @@
 {
-    "extends": "react-app",
-    "rules": {},
-    "overrides": [
-        {
-            "files": ["./src/util/ws_mousetrap_fork.js"],
```

Comment:
Is it OK that we are no longer overriding ws_mousetrap_fork.js? It's external code and we don't want it to be subject to linting.

At the same time, it looks like this PR is passing, so perhaps ws_mousetrap_fork.js is ignored anyway...?